ci/windows_ci_steps.sh

```diff
@@ -88,7 +88,7 @@ fi
 
 # Complete envoy-static build
 if [[ $BUILD_ENVOY_STATIC -eq 1 ]]; then
-  bazel "${BAZEL_STARTUP_OPTIONS[@]}" build "${BAZEL_BUILD_OPTIONS[@]}" //source/exe:envoy-static
+  bazel "${BAZEL_STARTUP_OPTIONS[@]}" build --output_groups=+pdb_file "${BAZEL_BUILD_OPTIONS[@]}" //source/exe:envoy-static
```

Comment:
New requirement to force bazel to download this when using remote exec

Reply:
i dont entirely understand why this is now required here but wondering if we need to add it anywhere else and/or update relevant docs

Reply:
cool, thanks for explanation